intake/interface/source/defined_plots.py

```diff
@@ -54,38 +59,62 @@ class Plots(BaseView):
     """
     select = None
 
-    def __init__(self, source=None, **kwargs):
-        self.custom = pn.widgets.Button(name='Customize...', width_policy='min', align='center')
+    def __init__(self, source=None, edit_callback=None, **kwargs):
+        self.custom = pn.widgets.Button(name='Create', width=70, align='center')
         self.source = source
+        self.callback_on_edit = edit_callback
         self.panel = pn.Column(name='Plot', width_policy='max', margin=0)
+        self._behavior_callback = None
         super().__init__(**kwargs)
 
     def setup(self):
         self.instructions = pn.pane.Markdown(
-            self.instructions_contents, align='center', width_policy='max')
+            "**Select Plot**", align='center', width=70)
         self.select = pn.widgets.Select(options=self.options, height=30,
                                         align='center', min_width=200)
+        self.edit_options = pn.widgets.Select(options=['  Edit', '  Clone', '  Rename', '  Delete'],
```

Comment:
Why the spaces in the labels?

Reply:
I'm creating a dropdown selector for additional options, but I want it rendered to look like a down arrow next to the "Edit" button. By adding spaces at the beginning and forcing `width=25`, it gets me what I want. It is a bit of a hack, but it works without requiring changes to panel.

I'll add a comment in the code because it isn't obvious why the spaces exist.